docs/hardware/pwm-receivers.md

```diff
@@ -5,7 +5,7 @@ description: ExpressLRS supports PWM output without the need of converters.
 
 ![HW Banner](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-hardware/master/img/hardware.png)
 
-ExpressLRS now supports direct PWM output from receivers that have been specifically designed for this mode. Any version-compatible TX firmware (e.g. 2.0.0 TX with 2.0.0 RX) works with PWM receivers-- only the RX firmware is specific to PWM. This documentation refers to only receivers with native PWM output, not receivers attached to external CRSF to PWM converters such as the [CRServoF](https://github.com/CapnBry/CRServoF/) or Matek CRSF-PWM-C.
+ExpressLRS now supports direct PWM output from receivers. This documentation refers to only receivers with native PWM output, not receivers attached to external CRSF to PWM converters such as the [CRServoF](https://github.com/CapnBry/CRServoF/) or Matek CRSF-PWM-C.
```

Comment:
Might as well add a link to the matek product page.

Reply:
Added